core/downsampling/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.ac.cam.downsampling</groupId>
```

Comment:
1. The group id is inconsistent with the project structure. Add a directory after the cares directory called downsampling (as mentioned in the review) and change the groupId to uk.ac.cam.cares.downsampling

Reply:
Thank you for the comment, I have updated accordingly.